docs/content/en/schemas/v1beta14.json

```diff
@@ -1229,67 +1194,39 @@
       "description": "patch to be applied by a profile.",
       "x-intellij-html-description": "patch to be applied by a profile."
     },
-    "JibGradleArtifact": {
+    "JibArtifact": {
       "properties": {
         "args": {
           "items": {
             "type": "string"
           },
           "type": "array",
-          "description": "additional build flags passed to Gradle.",
-          "x-intellij-html-description": "additional build flags passed to Gradle.",
+          "description": "additional build flags passed to the builder.",
+          "x-intellij-html-description": "additional build flags passed to the builder.",
           "default": "[]",
           "examples": [
             "[\"--no-build-cache\"]"
           ]
         },
-        "project": {
-          "type": "string",
-          "description": "selects which Gradle project to build.",
-          "x-intellij-html-description": "selects which Gradle project to build."
-        }
-      },
-      "preferredOrder": [
-        "project",
-        "args"
-      ],
-      "additionalProperties": false,
-      "description": "*alpha* builds images using the [Jib plugin for Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin).",
-      "x-intellij-html-description": "<em>alpha</em> builds images using the <a href=\"https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin\">Jib plugin for Gradle</a>."
-    },
-    "JibMavenArtifact": {
-      "properties": {
-        "args": {
-          "items": {
-            "type": "string"
-          },
-          "type": "array",
-          "description": "additional build flags passed to Maven.",
-          "x-intellij-html-description": "additional build flags passed to Maven.",
-          "default": "[]",
-          "examples": [
-            "[\"-x\", \"-DskipTests\"]"
-          ]
-        },
-        "module": {
+        "profile": {
```

Comment:
I feel like we should get rid of this and just translate it to `-Pprofile` in the `args` section?

Reply:
👍